consensus-types/blocks/proto.go

```diff
@@ -7,10 +7,10 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-// Proto returns the underlying protobuf signed beacon block.
+// Proto converts the signed beacon block to a protobuf object.
 func (b *SignedBeaconBlock) Proto() (proto.Message, error) {
 	if b == nil {
-		return nil, errNilBlock
+		return nil, nil
```

Comment:
Why I think this change is fine (especially `BeaconBlock.Proto()` and BeaconBlockBody.Proto()`:
1. Philosophically, converting a `nil` block to a `nil` proto equivalent makes sense.
2. Practically, `SignedBeaconBlock.Copy()` uses `Proto()` internally and will not work if the block or body are `nil`.